modules/cij/__init__.py

```diff
@@ -110,13 +111,15 @@ def expand_path(path):
 
     conf = {v: os.environ.get("_".join([prefix, v])) for v in names}
 
-    for env in (e for e in conf.keys() if e[:len(prefix)] in names and conf[e]):
+    for env in (e for e in conf.keys()
+                if e[:len(prefix)] in names and conf[e]):
         conf[env] = expand_path(conf[env])
         if not os.path.exists(conf[env]):
```

Comment:
I think the readability of this filter would be improved if it was assigned to a variable and given a name

Reply:
Agree, I've adjusted accrodingly.